src/ui/components/SearchInput/index.js

```diff
@@ -42,6 +42,9 @@ export default class SearchInput extends React.Component {
       // There is no transitionstart event, but animation will start if there is no value.
       this.setState({ animating: true });
     }
+    // Handle the possibility that onTransitionEnd will never fire if
+    // the CSS animation is a no-op due to the size of text in the label.
+    window.setTimeout(this.onTransitionEnd, 250);
```

Comment:
What about doing something like this?
````js
this.simulatedTransitionEnd = window.setTimeout(this.onTransitionEnd, 250);
````
That way the real `onTransitionEnd` can call `clearTimeout()` on it. I imagine without clearing it, the animation might look jumpy? But I don't know. If it won't make it jump then we probably don't need it.